source/guide_wkd.rst

```diff
@@ -0,0 +1,197 @@
+.. highlight:: console
+
+.. author:: Starkstromkonsument <it@starkstromkonsument.de>
+
+.. tag:: audience-admins
+.. tag:: mail
+.. tag:: self-hosting
+
+#######################
+Web Key Directory (WKD)
+#######################
+
+.. tag_list::
+
+[WKD] stands for "`Web Key Directory`_" and is a standard for making a users GnuPG / OpenPGP public key available via their e-mail provider or server with the domain that corresponds to their e-mail address. There's several clients (such as Enigmail in Thunderbird or OpenKeyChain on Android) that will use this standard to automatically fetch a user's public key, when writing an e-mail to them.
+
+Web Key Directories provide an easy way to discover public keys through HTTPS. They provide an important piece to the infrastructure to improve the user experience for exchanging secure emails and files. In contrast to the public keyservers a Web Key Directory does not publish mail addresses and it is an authoritative pubkey source for its domain.
+
+----
+
+.. note:: For this guide you should be familiar with the basic concepts of
+
+  * e-mail encryption with GnuPG / OpenPGP 
+  * :manual:`domains <web-domains>`
+
+Prerequisites
+=============
+
+You need the public keys for the e-mail identities you want to provide via WKD (e.g. isabell@stardust.uberspace.de). This guide does not explain how to generate GnuPG / OpenPGP keys.
```

Comment:
If you'd `isabell@uber.space` would make for a nicer and shorter example. That's an optional chnage, though :sweat_smile: